content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

```diff
@@ -135,24 +134,63 @@ Both of these issues can be resolved by configuring Istio to perform TLS origina
         name: https-port
         protocol: HTTPS
       resolution: DNS
-    ---
-    apiVersion: networking.istio.io/v1
-    kind: DestinationRule
-    metadata:
-      name: edition-cnn-com
-    spec:
-      host: edition.cnn.com
-      trafficPolicy:
-        portLevelSettings:
-        - port:
-            number: 80
-          tls:
-            mode: SIMPLE # initiates HTTPS when accessing edition.cnn.com
     EOF
     {{< /text >}}
 
-    The above `DestinationRule` will perform TLS origination for HTTP requests on port 80 and the `ServiceEntry`
-    will then redirect the requests on port 80 to target port 443.
+1.  Add a policy to perform TLS origination:
+
+{{< tabset category-name="tls-origination" >}}
+
+{{< tab name="Istio API" category-value="istio-api" >}}
+
+{{< text syntax=bash snip_id=apply_origination_destinationrule >}}
+$ kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1
+kind: DestinationRule
+metadata:
+  name: edition-cnn-com
+spec:
+  host: edition.cnn.com
+  trafficPolicy:
+    portLevelSettings:
+    - port:
+        number: 80
+      tls:
+        mode: SIMPLE # initiates HTTPS when accessing edition.cnn.com
+EOF
+{{< /text >}}
```

Comment:
These *need* to be indented to keep the format of the page and the numbering. This is an [existing issue](https://github.com/istio/istio.io/issues/15689) with tabset's partial. This cannot be left aligned like this to keep in styling used with the rest of the website.

Reply:
Ok, thanks. I reverted it.